features/tracker_board_view/lib/src/widgets/time_tracking_card/mobile_time_tracking_card.dart

```diff
@@ -1,27 +1,27 @@
 import 'package:core_ui/core_ui.dart';
+import 'package:domain/domain.dart';
 import 'package:flutter/material.dart';
 
-import '../../models/period_time_type.dart';
-import '../../models/tracker_card_type.dart';
-import '../hover_decorator.dart';
-import 'builders/time_description_builder.dart';
-import 'builders/tracker_card_background_color_builder.dart';
-import 'builders/tracker_card_icon_asset_name_builder.dart';
-import 'builders/tracker_card_title_builder.dart';
+import 'package:tracker_board_view/src/models/period_time_type.dart';
+import 'package:tracker_board_view/src/widgets/hover_decorator.dart';
+import 'package:tracker_board_view/src/widgets/time_tracking_card/builders/time_description_builder.dart';
+import 'package:tracker_board_view/src/widgets/time_tracking_card/builders/tracker_card_background_color_builder.dart';
+import 'package:tracker_board_view/src/widgets/time_tracking_card/builders/tracker_card_icon_asset_name_builder.dart';
+import 'package:tracker_board_view/src/widgets/time_tracking_card/builders/tracker_card_title_builder.dart';
 
 class MobileTrackerCard extends StatelessWidget {
-  final TrackerCardType trackerCardType;
-  final String currentHours;
-  final String previousHours;
+  final String categoryName;
+  final List<Activity> currentActivityList;
+  final List<Activity> previousActivityList;
   final PeriodTimeType periodTimeType;
 
   static const double _height = 140.0;
```

Comment:
Move to a separate file

Reply:
It is used only in his file, Why do I need to move it in a separate file?